<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Player</class>
 <widget class="QMainWindow" name="Player">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1097</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1097</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1097</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Player</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>1079</width>
      <height>505</height>
     </rect>
    </property>
    <widget class="VideoWidget" name="videoWidget" native="true">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>9</y>
       <width>781</width>
       <height>491</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="classBox">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>10</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>All</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Basketball</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Running</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Esport</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Soccer</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Others</string>
      </property>
     </item>
    </widget>
    <widget class="QCheckBox" name="timeBox">
     <property name="geometry">
      <rect>
       <x>940</x>
       <y>15</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Time</string>
     </property>
    </widget>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>50</y>
       <width>271</width>
       <height>451</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>269</width>
        <height>449</height>
       </rect>
      </property>
      <widget class="QListView" name="playlistView">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>271</width>
         <height>451</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>519</y>
      <width>1079</width>
      <height>112</height>
     </rect>
    </property>
    <widget class="QSlider" name="v_slider">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>791</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="volumn_slider">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>70</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="v_coverLabel">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>0</y>
       <width>31</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="v_labelDuration">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>14</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QToolButton" name="stopB">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="pauseB">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>60</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="preB">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="nextB">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>60</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="sB">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>60</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="oB">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>60</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>add</string>
     </property>
    </widget>
    <widget class="QToolButton" name="rB">
     <property name="geometry">
      <rect>
       <x>950</x>
       <y>60</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>remove</string>
     </property>
    </widget>
    <widget class="QToolButton" name="aB">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>60</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>open</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VideoWidget</class>
   <extends>QWidget</extends>
   <header>videowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>ClassChanged(int)</slot>
 </slots>
</ui>
